package.json

```diff
@@ -23,7 +23,7 @@
     ],
     "nohoist": [
       "**/@types/jest",
-      "**/@types/jasmine"
+      "**/@types/mocha"
```

Comment:
TypeScript validation fails with baffling errors if `@types/mocha` is hoisted; it loads Mocha globals (like `describe`) ignoring [`types`](https://www.typescriptlang.org/tsconfig/#types) / [`typeRoots`](https://www.typescriptlang.org/tsconfig/#typeRoots) settings. This breaks the build because Mocha's `describe` doesn't have `.each` extension present in Jest's `describe` and used in this PR to effortlessly test Ethereum compat.